cmd/auth/login.go

```diff
@@ -17,18 +17,16 @@ import (
 	"github.com/spf13/cobra"
 )
 
-func configureHost(ctx context.Context, persistentAuth *auth.PersistentAuth, args []string, argIndex int) error {
-	if len(args) > argIndex {
-		persistentAuth.Host = args[argIndex]
-		return nil
+func promptForProfile(ctx context.Context, dv string) (string, error) {
+	if !cmdio.IsInTTY(ctx) {
+		return "", fmt.Errorf("the command is being run in a non-interactive environment, please specify a profile using --profile")
 	}
 
-	host, err := promptForHost(ctx)
-	if err != nil {
-		return err
-	}
-	persistentAuth.Host = host
-	return nil
+	prompt := cmdio.Prompt(ctx)
+	prompt.Label = "Databricks profile name"
+	prompt.Default = dv
```

Comment:
`dv` -> `default`

Reply:
default is a reserved keyword in Go. Renamed this to `defaultValue`.